WORKSPACE

```diff
@@ -28,10 +28,10 @@ http_archive(
 #    and update the sha256 with the result.
 http_archive(
     name = "org_tensorflow",
-    sha256 = "579a74ad171d8da7b7193ff863f28482c2e6050c4090650b001fb80bbc46bb0f",
```

Comment:
(It's also quite likely that the TF version at head is now sufficiently new that you don't need the version bump any more, but you should verify that.)

Reply:
my version is a bit newer, titled "Unbreak non-GPU builds."

Reply:
Ok! Looks good.